### PR TITLE
Checkout page

### PR DESCRIPTION
**Pull request by amantulsyan35**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,13 +4,19 @@ import { Routes, Route } from 'react-router-dom';
 
 import Navbar from './components/Navbar/index';
 import Homepage from './Pages/Homepage';
+import Product from './Pages/Products';
+import Wishlist from './Pages/Wishlist';
+import Checkout from './Pages/Chceckout';
```
Comment (yaswanthmyneni) [inline src/App.js:9]: suggestion: create index.js file in pages and import all the pages from it.

File: src/Pages/Chceckout/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import './checkout.css';
```
Comment (yaswanthmyneni) [inline src/Pages/Chceckout/index.js:1]: UI: checkout page is overflowing.

**Review by yaswanthmyneni — APPROVED** [review]
You can merge after changes.

File: src/Pages/Chceckout/index.js (hunk 2 above)
Comment (amantulsyan35) [reply src/Pages/Chceckout/index.js:1]: Thanks will fix the bug

**Pull request closed without merging**